### ISSUE TITLE
[TECHDEBT] Get rid of the ChangeLog

### ISSUE THREAD

**Issue by jaakristioja** [bibletime/bibletime#366]
# Summary
We have kept a ChangeLog file in the repository, but keep forgetting to update it, hence it is does not accurately reflect what has actually changes. As discussed in #360 it no longer seems to serve a strong use-case.

# Consequences
It is a bother to update the file, and as we regularly forgetting to update it, makes it inaccurate and hence also misleading to users as well as developers.

# Remediation approach
ChangeLog should be removed from the master branch but kept in the stable-3.0 branch for future 3.0-series releases. Wiki pages referencing the ChangeLog should be updated.

**Comment by jaakristioja** [comment]
ChangeLog removed in 06f6b66a90a158ccb716f747d1ed68412ef94399.

**Comment by jaakristioja** [comment]
Wiki page about [Releasing BibleTime](https://github.com/bibletime/bibletime/wiki/Releasing-BibleTime) was updated by 845bcaf6af4ab2e641d6a87df6439fad35af4a0e:

```diff
commit 845bcaf6af4ab2e641d6a87df6439fad35af4a0e (HEAD -> master, origin/master, origin/HEAD)
Author: Jaak Ristioja <jaak@ristioja.ee>
Date:   Sun Jan 9 13:38:44 2022 +0200

    Releasing-BibleTime.md: Removed ChangeLog instructions, because ChangeLog was removed

diff --git a/Releasing-BibleTime.md b/Releasing-BibleTime.md
index e57acb3..e5545a9 100644
--- a/Releasing-BibleTime.md
+++ b/Releasing-BibleTime.md
@@ -15,12 +15,9 @@ Here are some easy to miss things which should be verified before each release.
 
 1.  In `/CMakeLists.txt` Adjust `BT_VERSION_MAJOR`, `BT_VERSION_MINOR` and `BT_VERSION_PATCH` (but do NOT yet set `BT_VERSION_BUILD`).
    -  Values for `BT_VERSION_PATCH` may include a `_rc#` or `_beta#` suffix where `#` is a positive number. 
-2. Add an entry to `/ChangeLog`, e.g:
-   - `2011-09-19 Your Name <your.name@mail.provider.com>`
-   - `    * Released 2.9 beta 1.`
 
 3. Do the commit:
-   - `git add CMakeLists.txt ChangeLog && git commit -m 'Released 2.9 beta 1'`
+   - `git add CMakeLists.txt && git commit -m 'Released 2.9 beta 1'`
 
 4. Tag and sign the commit (prepend a `v` to the tag name):
    - `git tag -s -m 'Released 2.9 beta 1' v2.9_beta1 HEAD`
```

All done. Closing.

**Comment by kalemas** [comment]
Flash)
